src/components/Notifications/ConnectedTimeConfig.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { Store } from 'redux';
+
+import { TimeConfigComponent } from './TimeConfig';
+
+export const ConnectedTimeConfig = (store: Store) => {
```

Comment:
It's a prop so you have to wrap it in object.
```suggestion
export const ConnectedTimeConfig = ({store}: {store: Store}) => {
```